### PR TITLE
Unmesh patch

### PR DESCRIPTION
**Pull request by unmeshvrije**
change homework for git for week 2 and week 3

### CODE REVIEW

File: Week2/MAKEME.md [hunk 1]
```diff
@@ -11,17 +11,22 @@
...
-- [Getting Started with Git and GitHub](https://www.codecademy.com/articles/f1-u3-git-setup)
-
-The next exercise is a short interactive course, that will take you through the basics of GIT usage:
-
-- [Learn GIT](https://www.codecademy.com/learn/learn-git)
-
-The final exercise is a mini-course of Git and GitHub. Go through it and try to code along:
-
-- [How to Use GIT and GitHub](https://eu.udacity.com/course/how-to-use-git-and-github--ud775)
+# Git branching homework:
```
Comment (NoerGitKat) [inline Week2/MAKEME.md:14]: Great exercise Unmesh! I like the step-by-step instructions.

File: Week3/MAKEME.md [hunk 2]
```diff
@@ -41,6 +45,12 @@ In order to speed up development it's wise to know how to use a CSS framework. I
 
 ### 3. Code along
 
+If you need more practice with Git, please follow these links and follow the instructions.
```
Comment (NoerGitKat) [inline Week3/MAKEME.md:48]: Perfect! Question: do you think tutorial #3 should still be concluded? (as you've mentioned you found it too advanced for students)

Comment (unmeshvrije) [reply Week3/MAKEME.md:48]: I think we can keep it, because some (smart) students might want to play with Git. We can make it optional.

**Review by unmeshvrije — COMMENTED** [review]
Should be possible to merge !

**Review by NoerGitKat — APPROVED** [review]
Thanks for your reply @unmeshvrije !

**Pull request merged**